lib/features/support/hooks.rb

```diff
@@ -172,6 +177,9 @@
     output_received_requests('logs')
   end
 
+  # Log all received requests to file
+  write_requests(scenario)
```

Comment:
There should be a way to turn this off

Reply:
Yeah, that's fair - an option defaulting to true ok though?

Reply:
One of the options has to be true or you'll get no failure reasons 😬